docker/build_scripts/build.sh

```diff
@@ -96,6 +97,8 @@ rm -rf sqlite-autoconf-3160200*
 # statically. We delete openssl afterwards.)
 build_openssl $OPENSSL_ROOT $OPENSSL_HASH
 mkdir -p /opt/python
+# Import public keys used to verify these signatures.
+gpg --import $MY_DIR/cpython-pubkeys.txt
```

Comment:
Actually please add the source URL (where those public keys where downloaded from) to the inline comment.

Reply:
I believe it's https://www.python.org/static/files/pubkeys.txt